Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:1.23.2-alpine
```

Comment:
This would technically only be the `build` stage. For the final "production" stage, there is no need to have `golang` as a base. We could do `alpine` or something smaller